src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -61,8 +61,35 @@ public static function debug_data() {
 			}
 		}
 
-		// Set up the array that holds all debug information.
-		$info = array();
+		/*
+		 * Set up the array that holds all debug information.
+		 *
+		 * When iterating through the debug data, the ordering of the sections
+		 * occurs in insertion-order of the assignments into this array. Setting
+		 * up empty values here preserves that specific ordering so it doesn't
+		 * depend on when inside this method each section is otherwise assigned.
+		 *
+		 * When all sections have been modularized, this will be the final single
+		 * assignment of the sections before filtering and none will be empty.
+		 *
+		 * @ticket 61648.
```

Comment:
```suggestion
		 * @ticket 61648
```
No need for the full stop, see https://core.trac.wordpress.org/ticket/61238.